package.json

```diff
@@ -30,10 +30,14 @@
     "url": "https://github.com/gxa/my-package.git"
   },
   "dependencies": {
+    "jest-cli": "^23.4.1",
```

Comment:
There are many new packages here. Remember to remove the ones you don’t use, move the ones that are used for development to `devDependencies`. The packages `webpack-command` and `webpack-serve` are replacements for `webpack-cli` and `webpack-dev-server`. Remove the latter.